maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -84,7 +84,7 @@ class Orchestra(
     private val onCommandStart: (Int, MaestroCommand) -> Unit = { _, _ -> },
     private val onCommandComplete: (Int, MaestroCommand) -> Unit = { _, _ -> },
     private val onCommandFailed: (Int, MaestroCommand, Throwable) -> ErrorResolution = { _, _, e -> throw e },
-    private val onCommandWarned: (Int, MaestroCommand) -> Unit = { _,  _ -> },
```

Comment:
There is a bit of auto-reformatting in this class. The main relevant change is `extractTextWithAICommand` method